src/Utilities/hooks/useEdgeGroups.js

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react';
+import useFeatureFlag from '../useFeatureFlag';
+import { fetchEdgeEnforceGroups } from '../../api';
+
+const useEdgeGroups = (value) => {
+  const [data, setData] = useState(value);
+  const edgeParityInventoryGroupsEnabled = useFeatureFlag(
+    'edgeParity.inventory-groups-enabled'
+  );
+
+  useEffect(() => {
+    if (edgeParityInventoryGroupsEnabled) {
```

Comment:
This is more of a nitpick, but theres no else chain off of this.
Instead we can do something like 

 `edgeParityInventoryGroupsEnabled &&
  (async () => {
        const response = await fetchEdgeEnforceGroups();
        const enforceEdgeGroups = response?.enforce_edge_groups;
        setData(enforceEdgeGroups);
      })();
    } 
    `
    within the useEffect.